recipes/gobject-introspection/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gobject-introspection" %}
+{% set major = "1.51" %}
+{% set patch = "1" %}
+{% set version = major ~ "." ~ patch %}
+{% set sha256 = "2c47e7632df936055b4e96f48315e2d392be10a0642ce11610fba5240e19772d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://ftp.gnome.org/pub/GNOME/sources/{{ name }}/{{ major }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - hugepaths.patch
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win or not py27]
```

Comment:
It looks like it should work on Windows, but frankly I don't need the Windows support and I'm not too keen to spend a lot of time getting it working right now. It can always be added later if the desire emerges.